tests/link/test_numba.py

```diff
@@ -8,7 +8,7 @@
 
 import aesara.scalar as aes
 import aesara.scalar.basic as aesb
-import aesara.scalar.basic_scipy as aes_sci
+import aesara.scalar.math as aes_sci
```

Comment:
This looks like it should be `import xxx as aesm` by looking at the others

Reply:
Sounds good. Will update.